jhu-idc/idc-isle-dc
# File download and restricted media access

This adds a restricted access field to all media types and the relevant UI changes to display a modal that allows downloading of media attached to a repository item. If restricted access is enabled then a user cannot download the file from the file system and should see no downlink in the download media modal.

To test:
- Run migrations for 10-backend-tests (or add relevant items and media manually)
- Visit repo items and click the download media button
- note the missing download link for media which are flagged as restricted and toggle the restricted flag to see a download link
